python/python/tests/test_transaction.py

```diff
@@ -658,6 +662,27 @@ def test_transaction_clear(self):
         transaction.clear()
         assert len(transaction.commands) == 0
 
+    @pytest.mark.parametrize("cluster_mode", [False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_standalone_copy_transaction(self, redis_client: RedisClient):
+        min_version = "6.2.0"
+        if not await check_if_server_version_lt(redis_client, min_version):
+            keyslot = get_random_string(3)
+            key = "{{{}}}:{}".format(
```

Comment:
Can we change to this so that the test is skipped if we're using a version less than 6.2.0?
```suggestion
        if not await check_if_server_version_lt(redis_client, min_version):
            return pytest.mark.skip(reason=f"Redis version required >= {min_version}")
        
        keyslot = get_random_string(3)
        key = "{{{}}}:{}".format(
```

Reply:
I will revert it back. I was trying to make changes to fix the CI/CD issue.